fineract-provider/src/main/resources/META-INF/spring.factories

```diff
@@ -0,0 +1,2 @@
+org.springframework.boot.autoconfigure.EnableAutoConfiguration=\
```

Comment:
Since this is part of the intermediate solution - while the modules are properly extracted - does it make sense to have this configured like this? The component scan would pick this up after all in fineract-provider. Thoughts?

Reply:
... yeah... was thinking the same when I put it in place... I guess technically we don't need it in this setup... probably my thinking was to put stuff in place and later move to separate modules; I guess there's an argument to keep only the bare minimum.
What I'll do: add a section in the documentation describing the anatomy of a clean module;  anyone who wants to port an existing or creating a new one has then a detailed todo list. Alright, working on it.